kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/topic/TopicDetails.java

```diff
@@ -7,24 +7,34 @@
 import lombok.experimental.ExtensionMethod;
 import org.openqa.selenium.By;
 
+import java.util.List;
+
 import static com.codeborne.selenide.Selenide.*;
 import static com.provectus.kafka.ui.utilities.WebUtils.clickByJavaScript;
 
 @ExtensionMethod({WaitUtils.class})
 public class TopicDetails {
 
+    protected SelenideElement loadingSpinner = $x("//*[contains(text(),'Loading')]");
     protected SelenideElement dotMenuBtn = $$x("//button[@aria-label='Dropdown Toggle']").first();
+    protected List<SelenideElement> navigationTabs = $$x("(//nav[@role='navigation'])[last()]/a");
```

Comment:
use Arrays.asList(bla-bla to use loop just in one place

Reply:
Refactored to using Arrays.asList